src/Engine/GameScenario.java

```diff
@@ -46,6 +46,19 @@ private TagMode(boolean multiCO)
     }
   };
 
+  public enum FogMode
+  {
+    OFF_TRILOGY(false, false), ON_TRILOGY(true, false), OFF_DOR(false, true), ON_DOR(true, true);
+
+    public final boolean fogDefaultsOn;
+    public final boolean dorMode;
+    private FogMode(boolean isOn, boolean DoR)
+    {
+      this.fogDefaultsOn = isOn;
+      this.dorMode = DoR;
+    }
```

Comment:
Whoah, are we saying we can't have two different kinds of fog active at the same time? I'm kinda surprised.

Reply:
I really can't justify that being intuitive from any direction, hah.
I think I did leave open a loophole in that local fog generation should be possible, but I'm not convinced I'll ever use it.